### PR TITLE
clash: add page

### PR DESCRIPTION
**Pull request by jackyliu16**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
1.14.0

### CODE REVIEW

File: pages/common/clash.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# clash
+
+> A rule-based tunnel in Go.
+> More information: <https://github.com/Dreamacro/clash/wiki>.
+
+- Specify a configuration [d]irectory:
+
+`clash -d {{path/of/configuration}}`
```
Comment (kbdharun) [inline pages/common/clash.md:8]:
```suggestion
`clash -d {{path/to/configuration}}`
```
or
```suggestion
`clash -d {{path/to/directory}}`
```

File: pages/common/clash.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+- Specify a configuration [f]ile:
+
+`clash -f {{path/of/configuration/file}}`
```
Comment (kbdharun) [inline pages/common/clash.md:12]:
```suggestion
`clash -f {{path/to/configuration_file}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution and welcome to tldr. I have some suggestions for this page.

**Review by kbdharun — APPROVED** [review]

**Review by mfrw — APPROVED** [review]
LGTM :)
Thanks @jackyliu16 for your contribution.

**Review by Magrid0 — APPROVED** [review]
Looks good to me, thanks for your contribution!

**Pull request merged**